src/map.h

```diff
@@ -1423,8 +1423,8 @@ class map
         void rad_scorch( const tripoint &p, const time_duration &time_since_last_actualize );
         void decay_cosmetic_fields( const tripoint &p, const time_duration &time_since_last_actualize );
 
-        void player_in_field( player &u );
-        void monster_in_field( monster &z );
+        void player_in_field( player &u, field &cur_field, bool inside = false );
+        void monster_in_field( monster &z, field &cur_field, bool inside = false );
```

Comment:
I don't see why `inside` needs to be optional. All calls to those functions provide an explicit value anyway, so why make it optional?